boards/openhwgroup/cv32a6_genesys_2/cv32a6_genesys_2_defconfig

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 CISPA Helmholtz Center for Information Security gGmbH
+# SPDX-License-Identifier: Apache-2.0
+CONFIG_BASE64=y
+CONFIG_INCLUDE_RESET_VECTOR=y
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_NS16550_ACCESS_WORD_ONLY=y
+CONFIG_CONSOLE_HANDLER=y
+CONFIG_XIP=n
+CONFIG_INIT_STACKS=y
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=25000000
+CONFIG_FPU=y
+
+# RNG
+CONFIG_TIMER_RANDOM_GENERATOR=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+# IRQs
+CONFIG_MULTI_LEVEL_INTERRUPTS=y
+CONFIG_2ND_LEVEL_INTERRUPTS=y
+# 1 PLIC
+CONFIG_NUM_2ND_LEVEL_AGGREGATORS=1
+CONFIG_PLIC=y
+CONFIG_3RD_LEVEL_INTERRUPTS=n
+
+# no networking support on this board
+
+# logging
+CONFIG_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=3
+CONFIG_THREAD_NAME=y
+
+# increased stack sizes
+CONFIG_ISR_STACK_SIZE=524288
+CONFIG_MAIN_STACK_SIZE=524288
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=524288
+CONFIG_IDLE_STACK_SIZE=524288
```

Comment:
yeah that's a no

Reply:
The board has 1 GB RAM. How large should the stacks be?

Reply:
Don't configure them at all, leave them at the default